client/packages/common/src/intl/locales/en/common.json

```diff
@@ -211,7 +211,7 @@
   "label.contact-tracing": "Contact Tracing",
   "label.contact-tracing-type": "Type of contact",
   "label.context": "Context",
-  "label.cost": "Cost",
+  "label.cost": "Pack cost price",
```

Comment:
Shouldn't we use a more descriptive `label.cost_per_pack` to make translation easier and have less mistakes?
I remember Adam raised something about french translations not having the correct details in them?

Reply:
yes! is requested in #4891